tests/utils/testutils.py

```diff
@@ -287,16 +295,18 @@ def _run_test(self, test, *args, **kwargs):
                 test(self, *args, **kwargs)
             except Exception as e:
                 test_exception = e
-
-            try:
-                self.host_stdout.seek(last_pos)
-                self.host_out = self.host_stdout.read()
-                self.host_stdout_logger.error(
-                    'Captured WebHost stdout from %s :\n%s',
-                    self.host_stdout.name, self.host_out)
             finally:
-                if test_exception is not None:
-                    raise test_exception
+                try:
+                    self.host_stdout.seek(last_pos)
+                    self.host_out = self.host_stdout.read()
+                    if self.host_out is not None:
```

Comment:
Does this check work? The failing 3.11 E2E tests printed "Captured WebHost log generated during test" but no other logs were printed

Reply:
updated.